org.flatpak.EasyEdit.yml

```diff
@@ -0,0 +1,90 @@
+id: org.flatpak.EasyEdit
```

Comment:
this app-id is invalid. You don't own `flatpak.org`

Reply:
Sorry did not know it worked that way, I followed instructions to build app and that is what is say the app I'd was. GitHub is where I post it no specific domain so what should the app I'd be?

Reply:
It's documented

https://docs.flathub.org/docs/for-app-authors/requirements#application-id